bin/dcos-commons-tools/airgap_linter.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
```

Comment:
Yes, I agree. This was partially to fix the recent changes in dcos-commons and it's also consistent with the rest of the standalone repos.